geoalchemy2/functions.py

```diff
@@ -323,10 +355,37 @@ def __init__(self, *args, **kwargs):
      'geomB. The geography implementation does a transform to geometry to do '
      'the intersection and then transform back to WGS84.'),
 
+    ('ST_LineInterpolatePoint', types.Geometry,
+     'Returns a point interpolated along a line. First argument must be a '
+     'LINESTRING. Second argument is a float8 between 0 and 1 representing '
+     'fraction of total linestring length the point has to be located.'
+     ''
+     'See ST_LineLocatePoint for computing the line location nearest to a Point.'),
```

Comment:
Please use the short description. See http://postgis.net/docs/ST_LineInterpolatePoint.html.